koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -58,6 +60,10 @@
 else:
     osbs_flatpak_support = True
 
+# root of koji_containerbuild module
+TOP_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+BUILD_OPTS_SCHEMA_PATH = os.path.join(TOP_DIR, 'schemas', 'build_opts.json')
```

Comment:
nitpick: should this file be packaged under a data directory instead?

Reply:
yeah that seems like a better place for it, done